src/server/protocol.ts

```diff
@@ -1617,6 +1617,7 @@ namespace ts.server.protocol {
          * Optional prefix to apply to possible completions.
          */
         prefix?: string;
+        includeCompletionsWithActions: boolean;

```

Comment:
i would rather we be more specific about what the feature does today, and make it `includeNonImportedDeclarations` or something along these lines.

Also please add a comment explaining what it does

Reply:
How about `includeExternalModuleExports`?

Reply:
👍